code/modules/security levels/security levels.dm

```diff
@@ -18,8 +20,16 @@
 			level = SEC_LEVEL_DELTA
 
 	//Will not be announced if you try to set to the same level as it already is
-	if(level >= SEC_LEVEL_GREEN && level <= SEC_LEVEL_DELTA && level != security_level)
+	if(level >= SEC_LEVEL_RAINBOW && level <= SEC_LEVEL_DELTA && level != security_level)
 		switch(level)
+			if(SEC_LEVEL_RAINBOW)
+				world << sound('sound/items/AirHorn.ogg')
+				//Attention! Code rainbow!
+				//There is an immediate serious lack of threats to the station. The clown may have bananas unholstered at all times. The Head of Personnel may now request additional entertainment staff.
+				var/dat = {"<font size=4><html><div><span style="color:#ff0000;">A</span><span style="color:#ff1900;">t</span><span style="color:#ff3300;">t</span><span style="color:#ff4c00;">e</span><span style="color:#ff6600;">n</span><span style="color:#ff7f00;">t</span><span style="color:#ff9f00;">i</span><span style="color:#ffbf00;">o</span><span style="color:#ffdf00;">n</span><span style="color:#ffff00;">!</span><span style="color:#ccff00;"> </span><span style="color:#99ff00;">C</span><span style="color:#66ff00;">o</span><span style="color:#33ff00;">d</span><span style="color:#00ff00;">e</span><span style="color:#00ff40;"> </span><span style="color:#00ff80;">r</span><span style="color:#00ffbf;">a</span><span style="color:#00ffff;">i</span><span style="color:#00ccff;">n</span><span style="color:#0099ff;">b</span><span style="color:#0066ff;">o</span><span style="color:#0033ff;">w</span><span style="color:#0000ff;">!</span></div></font>
```

Comment:
OH LAWD wouldn't it be easier to turn this into its own thing? span style="rainbow" or something

Reply:
hahaha oh jesus i didn't even see that
I'm agreeing with that.

Reply:
Quite literally wyci

I don't think it's doable to do what you're asking, or if it is it requires more advanced knowledge of html than I have.

Reply:
It'd be easy with CSS and using :nth-letter(#), even easier with Javashit

Reply:
Great, I'll rebase after someone else writes the css